Firebend.AutoCrud.CustomFields.Mongo/Implementations/MongoCustomFieldsCreateService.cs

```diff
@@ -138,9 +135,9 @@ private Task PublishUpdatedDomainEventAsync(TEntity previous,
         {
             Previous = previous,
             Operations = patch?.Operations,
-            EventContext = _domainEventContextProvider?.GetContext()
+            EventContext = domainEventContextProvider?.GetContext()
         };
 
-        return _domainEventPublisher.PublishEntityUpdatedEventAsync(domainEvent, entityTransaction, cancellationToken);
+        return domainEventPublisher.PublishEntityUpdatedEventAsync(domainEvent, entityTransaction, cancellationToken);
```

Comment:
i don't know how this was working before 😅  but shouldn't we have and `await` here?

Reply:
The method isn't marked with the async modifier so this should be fine